cmd/swagger/commands/generate/server.go

```diff
@@ -24,55 +24,60 @@ import (
 // Server the command to generate an entire server application
 type Server struct {
 	shared
-	Name              string   `long:"name" short:"A" description:"the name of the application, defaults to a mangled value of info.title"`
-	Operations        []string `long:"operation" short:"O" description:"specify an operation to include, repeat for multiple"`
-	Tags              []string `long:"tags" description:"the tags to include, if not specified defaults to all"`
-	Principal         string   `long:"principal" short:"P" description:"the model to use for the security principal"`
-	DefaultScheme     string   `long:"default-scheme" description:"the default scheme for this API" default:"http"`
-	Models            []string `long:"model" short:"M" description:"specify a model to include, repeat for multiple"`
-	SkipModels        bool     `long:"skip-models" description:"no models will be generated when this flag is specified"`
-	SkipOperations    bool     `long:"skip-operations" description:"no operations will be generated when this flag is specified"`
-	SkipSupport       bool     `long:"skip-support" description:"no supporting files will be generated when this flag is specified"`
-	ExcludeMain       bool     `long:"exclude-main" description:"exclude main function, so just generate the library"`
-	ExcludeSpec       bool     `long:"exclude-spec" description:"don't embed the swagger specification"`
-	WithContext       bool     `long:"with-context" description:"handlers get a context as first arg (deprecated)"`
-	DumpData          bool     `long:"dump-data" description:"when present dumps the json for the template generator instead of generating files"`
-	FlagStrategy      string   `long:"flag-strategy" description:"the strategy to provide flags for the server" default:"go-flags" choice:"go-flags" choice:"pflag"`
-	CompatibilityMode string   `long:"compatibility-mode" description:"the compatibility mode for the tls server" default:"modern" choice:"modern" choice:"intermediate"`
-	SkipValidation    bool     `long:"skip-validation" description:"skips validation of spec prior to generation"`
+	Name                   string   `long:"name" short:"A" description:"the name of the application, defaults to a mangled value of info.title"`
```

Comment:
line is 141 characters

Reply:
@fredbi why are you enforcing line lengths? It might make the code less readable.
According to [effective go](https://golang.org/doc/effective_go.html):
> Line length
Go has no line length limit. Don't worry about overflowing a punched card. If a line feels too long, wrap it and indent with an extra tab.

Reply:
it is a standard rule in golangci to improve readability.

If you find a nice way to tune this in .golangci.yml, so it does not take literal strings in its count, but otherwise works on regular statements, I am interested.

N.B: you may figure out locally using `golangci-lint run` (to be installed from github), which now replaces `gometalinter` on our repos...

Reply:
looks like the flag is `--disable lll`

Reply:
like mentioned above, I would prefer relaxing the check than disabling it

Reply:
ok. We discussed it and I rest my case. Let's disable it then